src/components/footer.tsx

```diff
@@ -1,22 +1,9 @@
 import type { NextPage } from 'next';
-import { CSSProperties } from 'react';
 
 const Footer: NextPage = () => {
-    const style = {
-        backgroundColor: 'blue',
-        color: 'white',
-        position: 'absolute',
-        width: '100%',
-        height: '30px',
-        bottom: 0,
-    } as CSSProperties;
-
     return (
-        <div className='footer' style={style}>
-            <div style={{ margin: '5px' }}>
-                CS: 498 - SAP
-            </div>
-        </div>
+        <footer>
```

Comment:
I'm curious, does this element already exist? I don't think it will render anything.

The footer can be deleted if we don't plan on using it, by the way.

Reply:
For your first point, I'd just change it to something like "Feat: added primary frontend pages"
Since this PR has become a bit of a monolith.
In the description, I'd also list all of the Jira tickets it addresses.

But to your second point about the footer, since it can easily be added later, we don't need to worry about subbing it for now.

Reply:
Sorry, forgot to explain editing PR names. There should be an edit button to the right of the name.

To edit the description, there should be the three dots in the right corner of the description box, click that and there should be an edit button.

Reply:
Renamed the PR.